### PR TITLE
Fix Editing the data file errors

### LINKED ISSUE #268: UG: Format Caution part
image

### PR DESCRIPTION
**Pull request by DennieDan**
Closes #268

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -374,7 +374,7 @@ _Details coming soon ..._
 ## FAQ
 
 **Q**: How do I transfer my data to another Computer?<br>
-**A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous AddressBook home folder.
+**A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous UGTeach home folder.
```
Comment (TY1Fan) [inline docs/UserGuide.md:378]: Nice catch! Thanks for correcting this bug!

**Review by TY1Fan — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 2]
```diff
@@ -351,15 +351,15 @@ Format: `exit`
 
 ### Saving the data
 
-AddressBook data are saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
+UGTeach data are saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
 
 ### Editing the data file
 
-AddressBook data are saved automatically as a JSON file `[JAR file location]/data/addressbook.json`. Advanced users are welcome to update data directly by editing that data file.
+UGTeach data are saved automatically as a JSON file `[JAR file location]/data/addressbook.json`. Advanced users are welcome to update data directly by editing that data file.
```
Comment (Feng1231) [inline docs/UserGuide.md:358]: `addressbook.json` missed out!

**Review by Feng1231 — CHANGES_REQUESTED** [review]
Please help to change the json file too! Thank you.

**Review by Feng1231 — APPROVED** [review]
LGTM

**Pull request merged**